en/docs/manage/api-management.md

```diff
@@ -0,0 +1,54 @@
+# Manage
+
+Choreo's API Management capabilities allow you to manage your APIs through the Choreo Console. You can manage the lifecycle, consumers, and documents, secure your backends by applying rate--limiting policies through usage plans, define permissions using scopes, configure security, rate-limiting, and visibility settings, and define any properties. 
+
+# Overview
+
+The Overview of the Manage section gives you a bird's eye view of how you currently manage your API component. You can view the lifecycle status of your component, the number of consumers, the security scheme configured, and the usage plan applied to the API. The overview pane also lists the environment(s) of the API.
```

Comment:
Shouldn't "The Overview of the Manage section" be "The **Overview** sub-tab of the **Manage** tab"?
 Each of those updates the console URL.
 
 What is the "overview pane"? Let's have the names of UI elements in bold.

Reply:
There are no tabs.. so it might be confusing.

Reply:
The overview pane is the sub-section of the Manage page.

Reply:
As discussed offline, let's discuss on it with the larger group and come back.